projects/update/client/updateProject.html

```diff
@@ -0,0 +1,8 @@
+<template name="updateProject">
+    <div class="page-header">
+      <h2>Update Project</h2>
+    </div>
+    
+    {{> quickForm collection="Projects" doc=getProject id="updateProjectForm" type="update"}}
```

Comment:
Remember to update the `doc=project` property, so that it uses the `project` template helper.